batch/itests/org.jboss.tools.batch.ui.itest/META-INF/MANIFEST.MF

```diff
@@ -41,15 +41,15 @@ Require-Bundle: org.junit,
  org.jboss.tools.common.ui,
  org.eclipse.jst.servlet.ui,
  org.eclipse.jst.j2ee,
- org.eclipse.sapphire.platform;bundle-version="0.7.1",
- org.eclipse.sapphire.modeling;bundle-version="0.7.1",
- org.eclipse.sapphire.modeling.xml;bundle-version="0.7.1",
- org.eclipse.sapphire.java;bundle-version="0.7.1",
- org.eclipse.sapphire.ui;bundle-version="0.7.1",
- org.eclipse.sapphire.ui.swt.xml.editor;bundle-version="0.7.1",
- org.eclipse.sapphire.ui.swt.gef;bundle-version="0.7.1",
- org.eclipse.sapphire.workspace,
- org.eclipse.sapphire.workspace.ui,
+ org.eclipse.sapphire.platform;bundle-version="9.0.0",
```

Comment:
We should be more specific in sapphire dependencies. The early 9.0.0.* versions of Sapphire have incompatible API. We should depend on 9.0.0.201505051659 or latter (need to double check that this version is correct)